src/test/ui/consts/const_limit/feature-gate-const_limit.rs

```diff
@@ -0,0 +1,12 @@
+#![const_limit="1"]
```

Comment:
Please also add a test actually hitting the limit (so the same test but with the feature gate enabled).